include/triton/core/tritonserver.h

```diff
@@ -1855,6 +1855,18 @@ TRITONSERVER_DECLSPEC struct TRITONSERVER_Error*
 TRITONSERVER_ServerOptionsSetCudaMemoryPoolByteSize(
     struct TRITONSERVER_ServerOptions* options, int gpu_device, uint64_t size);
 
+/// Set the size of the virtual address space that will be used
+/// for growable memory in implicit state.
+///
+/// \param options The server options object.
+/// \param gpu_device The GPU device to set the CUDA virtual address space size
+/// \param size The size of the CUDA virtual address space.
+/// \return a TRITONSERVER_Error indicating success or failure.
+TRITONSERVER_DECLSPEC TRITONSERVER_Error*
+TRITONSERVER_ServerOptionsSetCudaVirtualAddressSize(
```

Comment:
I believe because we have a singleton - we would only be able to set this once - for the future we would want to see if we can allow multiple server objects with different options (can be handled in the future).

Reply:
discussed offline will be part of future changes.